src/force/CabanaPD_ForceModels_PMB.hpp

```diff
@@ -252,6 +252,135 @@ auto createForceModel( PMB, Fracture, ParticleType particles,
         delta, K, G0, temp, alpha, temp0 );
 }
 
+template <typename TemperatureType>
+struct ForceModel<PMB, Elastic, DynamicTemperature, TemperatureType>
```

Comment:
I got confused about this. The heat transfer does not introduce a new mechanics model. What is this part for?